Makefile

```diff
@@ -80,6 +80,8 @@ install: build doc sdk doc-json
 	mkdir -p $(DESTDIR)$(LIBEXECDIR)
 	mkdir -p $(DESTDIR)$(OPTDIR)/debug
 	mkdir -p $(DESTDIR)/usr/bin
+	mkdir -p $(DESTDIR)/usr/libexec/xapi
+	mkdir -p $(DESTDIR)/etc
```

Comment:
Obsolete as the next line creates it.

Reply:
Yes, that is true. I had still added it explicitly because I was worried that someone might remove the line below as part of some refactor.